src/binding/BindingBase/index.ts

```diff
@@ -1,5 +1,5 @@
-import { IBindable } from '../../components/Element/index';
 import { Events, History, Observer } from '@playcanvas/observer';
+import { IBindable } from '../../components/Element/index';
```

Comment:
What's the rationale behind this? Alphabetical order, or absolute paths first, relative paths later?

Reply:
It's because I thought it best to put imports from external packages before imports for modules within the same package. I guess it's like in C++ where you would normally put system includes before relative includes from your own project.

Reply:
👍 That's what I thought, just needed to make sure there wasn't any other reason as well 🙂